webapp/src/Service/ExternalContestSourceService.php

```diff
@@ -620,7 +620,7 @@ public function importEvent(Event $event, array $eventsToSkip): void
 
         // Note the @vars here are to make PHPStan understand the correct types.
         $method = match ($event->type) {
-            EventType::AWARDS, EventType::TEAM_MEMBERS, EventType::ACCOUNTS, EventType::PERSONS => $this->ignoreEvent(...),
+            EventType::ACCOUNTS, EventType::AWARDS, EventType::MAP_INFO, EventType::PERSONS, EventType::TEAM_MEMBERS => $this->ignoreEvent(...),
```

Comment:
It would be better to have a catch-all for unknown events. According to the standard a system may always add its own event types and we can't foresee all of them.